hosts

```diff
@@ -0,0 +1,2 @@
+[mysql]
+mysql.opencraft.com ansible_host=149.202.171.102
```

Comment:
The hosts files could be on the private repo, though. It's not too important since this info will also be obtainable through DNS.